### PR TITLE
Fix schemaEnhancer support for default values on newly created accord…

### PR DESCRIPTION
**Pull request by sneridagh**
…ion blocks

### CODE REVIEW

File: src/components/manage/Blocks/Accordion/Edit.jsx [hunk 1]
```diff
@@ -182,8 +212,6 @@ const Edit = (props) => {
                     },
                   },
                 });
-              } else {
-                onChangeField(id, value);
```
Comment (avoinea) [inline src/components/manage/Blocks/Accordion/Edit.jsx:186]: @sneridagh Not sure why did you remove this. I think this is required when using together with volto-metadata-block

File: src/components/manage/Blocks/Accordion/Edit.jsx [hunk 2]
```diff
@@ -229,7 +257,7 @@ const Edit = (props) => {
         {!data?.readOnlySettings && (
           <BlockDataForm
             schema={accordionBlockSchema({ intl })}
-            title="Accordion block"
+            title="Accordion block 2"
```
Comment (avoinea) [inline src/components/manage/Blocks/Accordion/Edit.jsx:260]: Huh?

**Review by avoinea — CHANGES_REQUESTED** [review]

File: src/components/manage/Blocks/Accordion/Edit.jsx (hunk 1 above)
Comment (avoinea) [reply src/components/manage/Blocks/Accordion/Edit.jsx:186]: See https://github.com/eea/volto-accordion-block/pull/22/files

Comment (sneridagh) [reply src/components/manage/Blocks/Accordion/Edit.jsx:186]: Oh I didn't! I will chech the pr again

File: src/components/manage/Blocks/Accordion/Edit.jsx (hunk 2 above)
Comment (sneridagh) [reply src/components/manage/Blocks/Accordion/Edit.jsx:260]: Same, shit got in there

**Comment by sneridagh** [comment]
@avoinea now it looks better, I don't know what did I do with the changeset...

**Review by avoinea — APPROVED** [review]
